src/js/orders.spec.ts

```diff
@@ -39,11 +37,9 @@ export class Order {
 
   encode(): Buffer {
     const digest = this.getOrderDigest();
-    const { v, r, s } = ecsign(
-      Buffer.from(digest.slice(2), "hex"),
-      Buffer.from(this.wallet.privateKey.slice(2), "hex"),
-    );
-    return abi.rawEncode(
+    const signature = this.wallet._signingKey().signDigest(digest);
+    const { v, r, s } = utils.splitSignature(signature);
+    const encodedOrder = utils.defaultAbiCoder.encode(
```

Comment:
Much nicer, although seems unrelated to the PR.

Reply:
This was part of migrating to `buidler` test environment.